tensorflow_serving/tools/docker/Dockerfile.devel

```diff
@@ -56,7 +56,7 @@ RUN echo "build --spawn_strategy=standalone --genrule_strategy=standalone" \
     >>/root/.bazelrc
 ENV BAZELRC /root/.bazelrc
 # Install the most recent bazel release.
-ENV BAZEL_VERSION 0.2.0
+ENV BAZEL_VERSION 0.3.1
```

Comment:
I missed that you had this PR, but I already updated the BAZEL_VERSION to 0.3.0 (What TF uses) and added a WORKSPACE file check so that you get a useful error message right away.

You can revert that change, but keep the mock change above and we'll merge. Thanks!